docs/content/guidelines/design.mdx

```diff
@@ -0,0 +1,300 @@
+---
+title: Design guidelines
+---
+
+import {Caption, Do, Dont} from '@primer/gatsby-theme-doctocat'
+import {Grid, Flex, Box} from '@primer/components'
+
+These are guidelines to use as a reference when designing new Octicons. If you're looking for guidance on how to use Octicons in your design, see our [Usage guidelines](/guidelines/usage).
```

Comment:
Should _octicons_ be lowercase if in the middle of a sentence?

Reply:
Yeah, I think you're right.